src/visitpy/common/MethodDoc.C

```diff
@@ -4189,32 +4190,18 @@ const char *visit_GetPlotInformation_doc =
 "time queries that create multiple curves, e.g. Time Pick with multiple\n"
 "variables, the dictionary contains a 'Curves' object, and each curve\n"
 "is referenced by it's associated variable name. This was introduced in\n"
-"VisIt 3.4.1.\n"
-"\n"
-"\n"
-"Single Curve Example:\n"
-"\n"
-"#% visit -cli\n"
+"VisIt 3.4.1. Single Curve Example: #% visit -cli\n"
```

Comment:
@visit-dav/visit-developers I am wondering if anyone might recognize these changes to CLI docs and can point me to where they originate?

Reply:
https://github.com/visit-dav/visit/pull/19828

Reply:
Ok, I understand what was happening. I think both `MethodDoc.C` and `functions.rst` were updated but by manually changing each. When I then went to *regenerate* `MethodDoc.C` from `functions.rst`, I got garbled formatting in `MethodDoc.C` and couldn't understand why. I closed this PR and created a new one in which this issue was fixed.